packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -21,15 +27,22 @@ import MessageList from './messagelist';
 const CHAT_CLASS = 'dx-chat';
 const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 
-type Properties = ChatProperties & { title: string; showDayHeaders: boolean };
+type Properties = ChatProperties & {
+  title: string;
+  showDayHeaders: boolean;
+};
 
 class Chat extends Widget<Properties> {
   _chatHeader?: ChatHeader;
 
   _messageBox!: MessageBox;
 
+  _validationSummary!: ValidationSummary;
```

Comment:
[excess code?]
Isn't it excess together with import?

Reply:
Forgot remove after refactor. Thank you for your attentive